components.yaml

```
@@ -5,54 +5,54 @@ GEOSadas:
 env:
   local: ./@env
   remote: ../ESMA_env.git
-  tag: v1.4.3
+  tag: v3.2.1
   develop: main
 
 cmake:
   local: ./@cmake
   remote: ../ESMA_cmake.git
-  tag: v1.0.11
+  tag: v3.4.2
   develop: develop
 
 ecbuild:
   local: ./@cmake/@ecbuild
   remote: ../ecbuild.git
-  tag: geos/v1.0.0
+  tag: geos/v1.0.6
 
 NCEP_Shared:
   local: ./src/Shared/@NCEP_Shared
   remote: ../NCEP_Shared.git
-  tag: v1.1.0
+  tag: rt-GEOSadas-5_29_0_prep3
   develop: main
 
 GMAO_Shared:
   local: ./src/Shared/@GMAO_Shared
   remote: ../GMAO_Shared.git
-  tag: cvs/GEOSadas-5_27_1_p3
+  tag: rt-GEOSadas-5_29_0_prep3
```

```suggestion
  tag: v1.4.5
  develop: main
```
This should probably be moved to v1.4.5. This tag has everything from the **rt-GEOSadas-5_29_0_prep4** tag, as well as a newly-found IASI bug (https://github.com/GEOS-ESM/GMAO_Shared/pull/202), plus some other updates that shouldn't matter:

https://github.com/GEOS-ESM/GMAO_Shared/compare/rt-GEOSadas-5_29_0_prep4..v1.4.5

---

This is not correct. The file CMakeList.txt in GMAO_hermes in v1.4.5 wiped out cnv2prs.pl, why? What is echorc.pl? Why do we need this program when we have echorc.x ?

---

Uh. How do you mean? The file is still there:

https://github.com/GEOS-ESM/GMAO_Shared/blob/v1.4.5/GMAO_hermes/cnv2prs.pl

and in GMAO_Shared/GMAO_hermes/CMakeLists.txt it says:
```cmake
   install(PROGRAMS cnv2prs.pl ana5sfc echorc.pl DESTINATION bin)
```
(see https://github.com/GEOS-ESM/GMAO_Shared/blob/a1fe7192a4f9d0f7783acff0c8d12eef79f47298/GMAO_hermes/CMakeLists.txt#L76)

`cnv2prs.pl` is the first file in the `install()` line? So it will be installed as the file does exist. When you built this did the file not checkout or not install? If so then that means we have an *enormous* problem in our CMake system

As for `echorc.pl`, this came from @gmao-jstassi in https://github.com/GEOS-ESM/GMAO_Shared/pull/196 You would need to ask him about it. We took it in because we figured it was safe.

---

Note when I built this, I do see `cnv2prs.pl`. If you look on discover at:
```
/gpfsm/dswdev/mathomp4/Models/GEOSadas-5_29_0-with-MyChanges/GEOSadas/install-Release/bin/cnv2prs.pl
``` 
it is there and it seems to be the code I would expect to see. 

Did you install GEOSadas in a different way?